mariadb.html.md.erb

```diff
@@ -47,4 +47,19 @@ Cloud Foundry: [Spring Music Example](https://github.com/cloudfoundry-samples/sp
 
 ## <a id='limitations'></a> Known Limitations
 
-The base of our HA MariaDB offering is built by a galera cluster, which brings some [known limitations](https://mariadb.com/kb/en/mariadb/mariadb-galera-cluster-known-limitations/) with it.
+The base of our HA MariaDB offering is built by a Galera cluster, which brings some [known limitations](https://mariadb.com/kb/en/mariadb/mariadb-galera-cluster-known-limitations/) with it.
+
+Our Galera aaS is a shared Galera cluster. Each customer have their own Database (`CREATE DATABSE`) with rights inside their own database.
+
+## <a id='definer'></a> Definer and bind-service (Credentials changing)
+
+With `cf bind-service` the service broker creates new credentials (`cf env $APP` to view them). This will break Routines, Triggers, Views and Events because they still have the old user hardcoded.
```

Comment:
* should be `cf env APP_NAME` instead of `cf env $APP`
* "Routines, Triggers, Views and Events" should be "routines, triggers, views and events"
* I would write "because they are still using the old user" instead of "because they still have the old user hardcoded"